### PR TITLE
Homepage Issues Fix

### PR DESCRIPTION
**Pull request by subrat-dash-hcl**
Homepage Issues Fix

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/less/container.less [hunk 1]
```diff
@@ -68,7 +68,7 @@ header.header{
 		margin: 0;
 		
 		li{
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/less/container.less]: @subrat-dash-hcl Remove  li block

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productfilter/clientlibs/less/productfilter.less [hunk 2]
```diff
@@ -14,7 +14,7 @@
     top: 50px;
     width: 95% !important;
     background: #fff;
-    min-height: 60%;
+    //min-height: 60%;
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productfilter/clientlibs/less/productfilter.less]: @subrat-dash-hcl  Remove comment line

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/less/container.less (hunk 1 above)
Comment (subrat-dash-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/less/container.less]: Removed the li block

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productfilter/clientlibs/less/productfilter.less (hunk 2 above)
Comment (subrat-dash-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productfilter/clientlibs/less/productfilter.less]: updated @Rajamouli-hcl

**Comment by Rajamouli-hcl** [comment]
merged

**Pull request merged**